setup.py

```diff
@@ -41,7 +41,7 @@ def get_static_files(path):
     # https://pagure.io/python-daemon/issue/18
     'python-daemon<2.2.0',
     'python-dateutil==2.7.5',
-    'enum34>1.1.0;python_version<"3.4"',
+    'enum34>1.1.0;python_version<"3.4"'
```

Comment:
trailing commas in Python lists are valid. Are you saying pip has a problem with this?

Reply:
Yup.  Can't install luigi using pip starting from 2.8.0 using pip install luigi==2.7..9 for now

Reply:
This syntax is not supported well by older version of pip/setuptools. Not exactly sure which though.

Reply:
I meant python version constraint: `enum34>1.1.0;python_version<"3.4`, not the comma.

Reply:
I can confirm this is due to older version of setuptools.